index.html

```diff
@@ -4975,6 +4975,646 @@ <h2 id="AudioWorkerParamDescriptor">
+  osc.start();
+});
+            </pre>
+            <h4>
+              AudioWorkletGlobalScope: bitcrusher.js
+            </h4>
+            <pre>
+registerAudioWorkletProcessor('BitCrusher', class extends AudioWorkletProcessor {
+
+  static get parameterDescriptors () {
+    return [{
+      name: 'bitDepth',
+      defaultValue: 12,
+      minValue: 1
+      maxValue: 16 
+    }, {
+      name: 'frequencyReduction',
+      defaultValue: 0.5,
+      minValue: 0,
+      maxValue: 1
+    }]
+  }
+
+  constructor (options) {
+    // We don't need to look at options: only AudioParams are initialized,
+    // which were taken care of by the node.
+    super(options);
+    this.phase = 0;
+    this.lastSampleValue = 0;
+  }
+
+  process (inputs, outputs, parameters) {
+    var input = inputs[0];
+    var output = outputs[0];
+    var bitDepth = parameters.bitDepth;
+    var frequencyReduction = parameters.frequencyReduction;
+
+    for (var channel = 0; channel &lt; output.length; ++channel) { 
+      for (var i = 0; i &lt; output[channel].length; ++i) {
+        var step = Math.pow(0.5, bitDepth[i]);
+        this.phase += frequencyReduction[i];
+        if (this.phase &gt;= 1.0) {
+          this.phase -= 1.0;
+          this.lastSampleValue = 
+            step * Math.floor(input[channel][i] / step + 0.5);
+        }
+        output[channel][i] = this.lastSampleValue;
+      }
+    }
+  }
+
+});
+            </pre>
+          </section>
+          <section>
+            <h3>
+              VU Meter Node
+            </h3>
+            <p>
+              This example of a simple sound level meter further illustrates
+              how to create an <a><code>AudioWorkletNode</code></a> subclass
+              that acts like a native <a><code>AudioNode</code></a>, accepting
+              constructor options and encapsulating the inter-thread
+              communication (asynchronous) between
+              <a><code>AudioWorkletNode</code></a> and
+              <a><code>AudioWorkletProcessor</code></a> in clean method calls
+              and attribute accesses. This node does not use any output.
+            </p>
+            <h4>
+              Global Scope: vumeternode.js
+            </h4>
+            <pre>
+class VUMeterNode extends AudioWorkletNode {
+
+  constructor (context, options) {
+    // Setting default values for the input, the output and the channel count.
+    options.numberOfInputs = 1;
+    options.numberOfOutputs = 0;
+    options.channelCount = 1;
+
+    options.updatingInterval = options.hasOwnProperty('updatingInterval') 
+      ? options.updatingInterval 
+      : 100;
+
+    super(context, 'VUMeter', options);
+
+    // Mirrored states of AudioWorkletProcessor.
+    this.updatingInterval = options.updatingInterval;
+    this.volume = 0;
+  }
+
+  get updatingInterval() {
+    return this.updatingInterval;
+  }
+
+  set updatingInterval (intervalValue) {
+    this.updatingInterval = intervalValue;
```

Comment:
Minor nitpick: won't setting the value of `updatingInterval` within a setter of the same name lead to infinite recursion? Likewise, a few lines up, getting the value of `updatingInterval` within a getter of the same name  could also result in a recursion error.